config/routes.rb

```diff
@@ -2,11 +2,15 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html\
 
   root 'welcome#index'
+  resources :users, only: [:new, :create]
 
-  resource :dashboard, controller: 'user', only: :show do
+  get '/login', to: 'users#login_form'
+  post '/login', to: 'users#login'
```

Comment:
Great job at getting the login function working here. Making a note to have these routes be more restful, we can create a `sessions_controller` that handles the `log in` and `log out` routes